### ISSUE TITLE
BatchNorm

### ISSUE THREAD

**Issue by cbasavaraj** [titu1994/keras-SRU#3]
Hi @titu1994, do you think your SRU could improve further with the use of BatchNorm as in Cooijmans et al. (2017), which has been implemented by @jihunchoi here for an LSTM?
Thank you.

**Comment by titu1994** [comment]
I'll have to give the paper a look. It may.be possible to implement it probably.
